s3-trigger-stack.yaml

```diff
@@ -79,9 +79,32 @@ Resources:
               response = {}
               try:
                   if event['RequestType'] == 'Delete':
-                      cfnresponse.send(event, context, cfnresponse.SUCCESS, {})
-                      return {}
-                  
+                      try:
+
+                          event_data = event['ResourceProperties']
+                          lambda_arn = event_data.get('LambdaFunctionArn', '')
+                          buckets = event_data.get('S3BucketArns', [])
```

Comment:
you can move it outside if branch as its common for delete and create.

Reply:
Done